<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1105</width>
    <height>649</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Bildorientierung korrigieren</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout" stretch="1,0,0">
        <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
         <widget class="QLabel" name="original_label">
          <property name="layoutDirection">
           <enum>Qt::LeftToRight</enum>
          </property>
          <property name="text">
           <string>Originales Bild</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QPushButton" name="minusButton_2">
            <property name="text">
             <string>-90°</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSlider" name="horizontalSlider">
            <property name="minimum">
             <number>-180</number>
            </property>
            <property name="maximum">
             <number>180</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="plusButton_2">
            <property name="text">
             <string>+90°</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item alignment="Qt::AlignHCenter">
         <widget class="QPushButton" name="correct_rotation_button">
          <property name="text">
           <string>Bildorientierung korrigieren</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2" stretch="1,0,0">
        <item alignment="Qt::AlignHCenter">
         <widget class="QLabel" name="rotated_label">
          <property name="text">
           <string>Rotiertes Bild</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,0,0">
          <property name="spacing">
           <number>2</number>
          </property>
          <property name="sizeConstraint">
           <enum>QLayout::SetDefaultConstraint</enum>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <item alignment="Qt::AlignLeft">
           <widget class="QPushButton" name="minusButton">
            <property name="text">
             <string>-90°</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSlider" name="horizontalSlider_2">
            <property name="minimum">
             <number>-180</number>
            </property>
            <property name="maximum">
             <number>180</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item alignment="Qt::AlignRight">
           <widget class="QPushButton" name="plusButton">
            <property name="text">
             <string>+90°</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item alignment="Qt::AlignHCenter">
         <widget class="QPushButton" name="save_button">
          <property name="text">
           <string>Bild Speichern</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item alignment="Qt::AlignHCenter">
     <widget class="QPushButton" name="autocorrectButton">
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>Mehrere Bilder korrigieren</string>
      </property>
      <property name="iconSize">
       <size>
        <width>16</width>
        <height>20</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>5</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1105</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuDatei">
    <property name="title">
     <string>Datei</string>
    </property>
    <addaction name="open_button"/>
   </widget>
   <widget class="QMenu" name="menuTheme">
    <property name="title">
     <string>Theme</string>
    </property>
    <addaction name="actionDark_mode"/>
    <addaction name="actionLight_mode"/>
    <addaction name="actionAuto"/>
   </widget>
   <addaction name="menuDatei"/>
   <addaction name="menuTheme"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="open_button">
   <property name="text">
    <string>Bild Öffnen</string>
   </property>
  </action>
  <action name="actionDark_mode">
   <property name="text">
    <string>Dark</string>
   </property>
  </action>
  <action name="actionLight_mode">
   <property name="text">
    <string>Light</string>
   </property>
  </action>
  <action name="actionAuto">
   <property name="text">
    <string>Auto</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
